src/select-field.jsx

```diff
@@ -141,6 +141,8 @@ const SelectField = React.createClass({
       hintText,
       fullWidth,
       errorText,
+      onFocus,
```

Comment:
@shaurya947 Thanks for the input, I've removed those handlers from `DropDownMenu`.

@oliviertassinari The docs for `SelectField` appear to have `onFocus` and `onBlur` written up on current `master` so I'm not editing the docs.

Is there anything else you'd like done before merge?